package.json

```diff
@@ -14,7 +14,7 @@
     "docs-build-schema": "./bin/build_db_schema.sh",
     "docs-clear": "(cd website && npm run clear)",
     "docs-serve": "npm run docs-build && (cd website && npm run start)",
-    "eslint": "eslint --ext .js --ext .ts .",
+    "eslint": "eslint --ext .js --ext .ts . --quiet",
```

Comment:
This could be contentious but we have 3k+ warnings, which is too many for the default console length. Any objections to quieting the warnings?

Reply:
I'm fine with this, maybe, but I think we need team buy-in.